pulpcore/cli/rpm/repository.py

```diff
@@ -165,16 +166,49 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
 @name_option
 @href_option
 @remote_option
-@click.option("--mirror/--no-mirror", default=None)
 @click.option(
-    "--skip-type", "skip_types", multiple=True, type=click.Choice(SKIP_TYPES, case_sensitive=False)
+    "--mirror/--no-mirror",
+    default=None,
+    help="""
+    DEPRECATED: If True, sync_policy will default to 'mirror_complete' instead
+    of 'additive'.
+    """,
+)
+@pulp_option(
+    "--optimize/--no-optimize",
+    default=None,
+    help="Whether or not to optimize sync.",
+    needs_plugins=[PluginRequirement("rpm", "3.3.0")],
```

Comment:
Tiny little bit more explicit:
```suggestion
    needs_plugins=[PluginRequirement("rpm", min="3.3.0")],
```